README.rst

```diff
@@ -71,6 +71,84 @@ Python versions::
 
     $ tox
 
+
+CRUD
+----
+
+Create
+~~~~~~
+
+.. code:: python
+
+    import pymysql
+    connection = pymysql.connect(host='host',
+                                 user='user',
+                                 passwd='passwd',
+                                 db='db')
+    with closing(connection.cursor()) as cursor:
+        sql = ("CREATE TABLE `users` ("
+               "`id` int(11) NOT NULL, "
```

Comment:
As far as I know, "C" in "CRUD" is "Creating new record". It means "INSERT", not "CREATE TABLE".